pkg/controllers/leaderworkerset_controller.go

```diff
@@ -569,3 +573,12 @@ func templateUpdated(sts *appsv1.StatefulSet, lws *leaderworkerset.LeaderWorkerS
 func replicasUpdated(sts *appsv1.StatefulSet, lws *leaderworkerset.LeaderWorkerSet) bool {
 	return *sts.Spec.Replicas != *lws.Spec.Replicas
 }
+
+func getLastTransitionTime(conditionType string, lws *leaderworkerset.LeaderWorkerSet) metav1.Time {
```

Comment:
maybe move this to a util file?

Reply:
Was thinking about that as well, can create an lws utils file similar to the one that exists for pods, and move this function, `templateUpdated` and `replicasUpdated` to it in a different PR.